dataeng/resources/dbt-automated.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup to run dbt commands
+cd $WORKSPACE/warehouse-transforms
+
+# Using --first-parent flag helps to avoids any intermediate commits that developers sometimes leave and merge without squashing. It helps us ignoring the noise in commit history.
+HEAD_COMMIT=$(git log --first-parent origin/master --format='%H' --max-count=1)
+
+if git log --format=%B -n 1 $HEAD_COMMIT | grep 'Schema Builder automated dbt update at' -q; then isSchemaBuilderPR="true"; else isSchemaBuilderPR="false"; fi
+
```

Comment:
Keeps polling for warehouse-transform repository and trigger when new changes are merged, then this conditions checks if PR message or title contains `Schema Builder automated dbt update at` and this is how it detects if this is one of Schema Builder generated PR which is merged

Reply:
Writing just for more clarity that if team member/on-call person use "Create a Merge commit" option to merge PR then `Schema Builder automated dbt update at` will be a commit message however if team member use "Squash commit" option to merge PR then `Schema Builder automated dbt update at` will be commit title.

`if git log --format=%B -n 1 $HEAD_COMMIT` tackles both the scenarios